disnake/interactions/base.py

```diff
@@ -272,6 +274,16 @@ def permissions(self) -> Permissions:
             return Permissions(self._permissions)
         return Permissions.private_channel()
 
+    @property
+    def app_permissions(self) -> Permissions:
+        """:class:`Permissions`: The resolved permissions of ourselves in the channel, including overwrites.
+
+        .. versionadded:: 2.6
+        """
+        if self.guild_id:
```

Comment:
missing a comment explaining why it works like this, and doesn't just always use `self._app_permissions`

Reply:
I didn't add a comment because its now explained by the docstring https://github.com/DisnakeDev/disnake/pull/586/commits/876b4d67200f85a8d4e6a8713652d5cc12d579e3